components/commercial_filter/views/popular_artists/popular_artists_view.coffee

```diff
@@ -8,7 +8,7 @@ module.exports = class PopularArtistsView extends Backbone.View
     @listenTo @artists, 'reset', @render
 
   render: ->
-    if @params.get('medium') in ['design', 'jewelry']
+    if @params.get('medium') in ['design', 'jewelry'] or @params.get('page') != 1
```

Comment:
We hide the widget on subsequent pages, as well as those two medium filters (since the results aren't great).